src/EditorFeatures/Test/SolutionCrawler/WorkCoordinatorTests.cs

```diff
@@ -1223,6 +1280,12 @@ public Task AnalyzeDocumentAsync(Document document, SyntaxNode bodyOpt, Invocati
             #endregion
         }
 
+        [ExportWorkspaceService(typeof(IGlobalOperationNotificationService), SolutionCrawler), Shared, PartNotDiscoverable]
```

Comment:
❕ This can be removed now. The default implementation of `IGlobalOperationNotificationService` in the master branch already does this.